src/commands/test/wizard.ts

```diff
@@ -0,0 +1,295 @@
+import * as fs from "fs";
+import * as os from "os";
+import * as path from "path";
+import { prompt, out } from "../../util/interaction";
+import { Questions } from "inquirer";
+import { help, CommandArgs, AppCommand, CommandResult } from "../../util/commandline";
+import { Messages } from "./lib/help-messages";
+import { AppCenterClient } from "../../util/apis";
+import { DeviceSet, DeviceConfiguration, AppResponse } from "../../util/apis/generated/models";
+import { DeviceConfigurationSort } from "./lib/deviceConfigurationSort";
+import RunEspressoWizardTestCommand from "./lib/wizard/espresso";
+import RunAppiumWizardTestCommand from "./lib/wizard/appium";
+import RunUitestWizardTestCommand from "./lib/wizard/uitests";
+import RunXCUIWizardTestCommand from "./lib/wizard/xcuitest";
+import { fileExistsSync } from "../../util/misc";
+
+enum TestFramework {
+  "Espresso" = 1,
+  "Appium" = 2,
+  "XCUI" = 3,
+  "Xamarin" = 4,
```

Comment:
I would change this to be either Xamarin.UITest or just UITest.

Reply:
Agree. @annakocheshkova please change to Xamarin.UITest for clarity.